### ISSUE TITLE
Log-In Issue

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#9]
```
What steps will reproduce the problem?
1. Type in Name and Password
2. Click Connect

What is the expected output? What do you see instead?
Expected: System logs in and inventory becomes available.
Actual: System freezes just after avatar is logged in and does not allow
any operator actions.

What version of the product are you using? On what operating system?
Version: 2.7.0.0  OS: Windows Vista Home Premium
```

Original issue reported on code.google.com by `orbit...@gmail.com` on 27 Oct 2008 at 5:39

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
FIXED: Issue 9: Doing two SL Connections in a row causes freezing
This issue has been FIXED in version 2.10.16
```

Original comment by `joesox` on 23 Jan 2009 at 7:45
- Changed state: **Fixed**
